src/ui/settings/general-settings-ui-handler.ts

```diff
@@ -3,18 +3,41 @@ import { hasTouchscreen, isLandscapeMode } from "#app/utils";
 import { t } from "i18next";
 import { AbstractSettingsUiHandler } from "./abstract-settings-ui-handler";
 
-export class SettingsUiHandler extends AbstractSettingsUiHandler {
+export class GeneralSettingsUiHandler extends AbstractSettingsUiHandler {
+  private onWindowResizeEvent = () => this.updateMoveTouchControlsSettingsLabel();
```

Comment:
NIT: alternatively use:
```suggestion
  private onWindowResizeEvent = this.updateMoveTouchControlsSettingsLabel.bind(this);
```

No difference, just giving insight

Reply:
interesting, thank you 📝 

This was done this way initially as I was following a few other components/handlers already doing the same. So for consistency right now i'll probably leave it as is